packages/monorepo-scripts/src/convert_changelogs.ts

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+
+import * as fs from 'fs';
+import lernaGetPackages = require('lerna-get-packages');
+import * as _ from 'lodash';
+import * as moment from 'moment';
+import * as path from 'path';
+
+import { Changelog, Changes, UpdatedPackage } from './types';
+import { utils } from './utils';
+
+const MONOREPO_ROOT_PATH = path.join(__dirname, '../../..');
+
+(async () => {
+    const allLernaPackages = lernaGetPackages(MONOREPO_ROOT_PATH);
+    const publicLernaPackages = _.filter(allLernaPackages, pkg => !pkg.package.private);
+    _.each(publicLernaPackages, lernaPackage => {
+        const changelogMdIfExists = getChangelogMdIfExists(lernaPackage.package.name, lernaPackage.location);
+        if (_.isUndefined(changelogMdIfExists)) {
+            throw new Error(`${lernaPackage.package.name} should have CHANGELOG.md b/c it's public. Add one.`);
+        }
+
+        const lines = (changelogMdIfExists as any).split('\n');
```

Comment:
`as string`. And it's strange, that TS doesn't detect it already being checked for undefined